CLAUDE.md

```
@@ -7,6 +7,41 @@
+## BrainBar Stub Warnings
+
+BrainBar Swift daemon has 4 STUB tools returning fake success:
+- brain_digest, brain_update, brain_expand, brain_tags — ALL BROKEN
+- Working: brain_search, brain_store, brain_recall, brain_entity
+- Last successful digest: March 14, 2026
+
+## Compact Instructions
+
+When compacting this session, follow these rules strictly:
+
+### NEVER preserve
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)